fhir/scripts/populate.js

```diff
@@ -0,0 +1,149 @@
+const asyncHandler = require('../src/lib/async-handler');
+const { COLLECTION } = require('../src/constants');
+const { mongoConfig } = require('../src/config');
+const package = require('../package.json');
+const mongo = require('../src/lib/mongo');
+const program = require('commander');
+const path = require('path');
+const glob = require('glob');
+
+// const {
+// 	PatientSchema,
+// 	ObservationSchema
+// } = require('fhir-schemas');
+
+// const SCHEMA_MAP = {
+// 	'Patient': PatientSchema,
+// 	'Observation': ObservationSchema
+// };
+
+const VALID_PROFILE_NAMES = [
+	COLLECTION.OBSERVATION,
+	COLLECTION.PATIENT
+];
+
+/**
+ * @function loadProfiles
+ * @summary Load the profiles into Mongo and log status as we go
+ */
+let loadProfiles = async args => {
+	let { profiles, reset } = args;
+	// Connect to Mongo
+	let [ err, client ] = await asyncHandler(mongo(mongoConfig.connection, mongoConfig.options));
+
+	// Fail if ther is an error
+	if (err) {
+		console.error(err);
+		process.exit(err);
+	}
+
+	// Connect to our Database
+	let db = client.db(mongoConfig.db_name);
+	let collectionList = await db.listCollections().toArray();
+	let currentCollections = collectionList.map(coll => coll.name);
+
+	// Iterate over our profiles and insert  all of our documents
+	let profile_names = Object.keys(profiles);
+	for (let name of profile_names) {
+		let documents = profiles[name];
```

Comment:
Where do you define the schema for the mongo collection?  Or does it not need one?

Reply:
We don't need one, we are not using mongoose and the current fhir-schemas repo needs to be updated before it can be used.